src/include/platform/OTARequestorInterface.h

```diff
@@ -133,10 +133,48 @@ class ProviderLocationList
         return CHIP_ERROR_NOT_FOUND;
     }
 
+    /**
+     * Load the DefaultOTAProvidersList attribute value from the platform KVS
+     * TODO: Use DefaultStorageKeyAllocator.AttributeValue() to get a key composed of the concrete attribute path
```

Comment:
If we ever support having the OTARequestor cluster across multiple endpoints in the same device, we can use the DefaultStorageKeyAllocator.AttributeValue() to get a key comprising of the full attribute path. Should be a simple change. For now we simply have one KVS value across all endpoints.

Reply:
I think we should use `DefaultStorageKeyAllocator` anyway, to keep all KVS keys in a single place.